GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -58,6 +59,19 @@ protected override void Execute(GVFSEnlistment enlistment)
                         { nameof(this.EnlistmentRootPathParameter), this.EnlistmentRootPathParameter },
                     });
 
+                if (this.MaintenanceJob != null

```

Comment:
Maybe add a comment here that this code is only intended to be run by the functional tests?